packages/core/src/utils/string-utils.ts

```diff
@@ -31,6 +38,32 @@ export function deserialize<T>(stringified: string): T {
   return JSON.parse(stringified);
 }
 
+export function getEmojiForStatus(status: schema.MutantStatus): string {
+  switch (status) {
+    case MutantStatus.Killed:
+      return '✅';
+    case MutantStatus.NoCoverage:
+      return '🙈';
+    case MutantStatus.Ignored:
+      return '🤥';
+    case MutantStatus.Survived:
+      return '👽';
+    case MutantStatus.Timeout:
+      return '⌛';
+    case MutantStatus.RuntimeError:
+    case MutantStatus.CompileError:
+      return '💥';
+  }
+}
+
+export function inputLength(input: string): number {
```

Comment:
I've thought about this name, probably `stringWidth` is a better name here. It communicates better the difference from the `.length` property.